src/components/research-form-wizard/FormPages/FollowUp.tsx

```diff
@@ -0,0 +1,214 @@
+import { useResearchWizardStore } from '@/stores/researchWizard';
+import { isValidEmail } from '@/utils/form-validation';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 400px;
+  background: white;
+  border-radius: 10px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+`;
+
+const Header = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+`;
+
+const Title = styled.h2`
+  margin: 0;
+`;
+
+const CloseButton = styled.button`
+  background: none;
+  border: none;
+  font-size: 20px;
+  cursor: pointer;
+`;
+
+const Description = styled.p`
+  font-size: 14px;
+  color: #555;
+`;
+
+const SubTitle = styled.p`
+  font-weight: bold;
+`;
+
+const RadioGroup = styled.div`
+  display: flex;
+  gap: 20px;
+  margin: 20px 0;
+`;
+
+const RadioLabel = styled.label`
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  cursor: pointer;
+`;
+
+const RadioInput = styled.input`
+  appearance: none;
+  border: 1px solid lightgray;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  position: relative;
+  cursor: pointer;
+
+  &:checked {
+    border-color: #6e56cf;
+    background-color: #aa99ec;
+  }
+
+  &:checked::before {
+    content: '';
+    position: absolute;
+    top: 4px;
+    left: 4px;
+    width: 10px;
+    height: 10px;
+    background-color: white;
+    border-radius: 50%;
+  }
+`;
+
+const EmailInput = styled.input<{ visible: boolean }>`
+  display: ${(props) => (props.visible ? 'block' : 'none')};
```

Comment:
Let's use `$` to avoid adding this properties to DOM
```suggestion
  display: ${(props) => (props.$visible ? 'block' : 'none')};
```